rfc6962-bis.xml

```diff
@@ -835,6 +858,9 @@ signatures.
         <t>
           <spanx style="verb">root_hash</spanx> is the root of the Merkle Hash Tree.
         </t>
+        <t>
+          <spanx style="verb">extensions</spanx> are future extensions to STH v2. Currently, no extensions are specified. If an implementation sees an extension that it does not understand, it SHOULD ignore that extension. Furthermore, an implementation MAY choose to ignore any extension(s) that it does understand.
```

Comment:
I presume this matches the SCT text, but I wonder ... should we leave it to the extension whether it can be ignored if understood? Though I guess in practice its up to the implementor to decide what is "understood".